tests/acceptance/ContactManagementCest.php

```diff
@@ -0,0 +1,209 @@
+<?php
+
+use Page\Acceptance\ContactPage;
+use Step\Acceptance\Contact;
+
+class ContactManagementCest
+{
+    public function _before(AcceptanceTester $I)
+    {
+        $I->login('admin', 'Maut1cR0cks!');
+    }
+
+    public function createContactFromQuickAdd(
+        AcceptanceTester $I,
+        Contact $contact
+    ): void {
+        $I->amOnPage(ContactPage::$URL);
+
+        // Click on "Quick Add" button
+        $I->waitForElementClickable(ContactPage::$quickAddButton, 30);
+        $I->click(ContactPage::$quickAddButton);
+
+        // Wait for the Quick Add Form to appear
+        $I->waitForElementVisible(ContactPage::$quickAddModal, 30);
+        $I->see('Quick Add', 'h4.modal-title');
+
+        $contact->fillContactForm('QuickAddFirstName', 'QuickAddLastName', 'quickadd@example.com', 'TestTag');
+
+        // Submit the form
+        $I->waitForElementClickable(ContactPage::$saveButton, 30);
+        $I->click(ContactPage::$saveButton);
+        $I->waitForElementNotVisible(ContactPage::$quickAddModal, 30);
+
+        $I->reloadPage(); // Ensure the latest data is loaded
```

Comment:
This shouldn't be necessary. The Ajax call reloads the page automatically. If it requires a hard reload then it is broken.